integration_testing/features/super-admin/super-admin-bulk-import-users.feature

```diff
@@ -1,28 +1,35 @@
-Feature: Super Admin import users into a facility in a device with manage bulkexportusers command
-    Super Admin needs to be able to do a bulk import of the users and classes, with the information to enroll or assign coaches to the classes
+Feature: Super Admin bulk import users into a facility
+    Super Admin needs to be able to do a bulk import of users and classes using the command line
```

Comment:
I think that stating that the enroll or assign information is important. This is the most important difference between this command and the existing import users command.

Reply:
Good point, wasn't thinking about that aspect, will add back! :+1: